runtime/common/src/lib.rs

```diff
@@ -1477,8 +1466,16 @@ macro_rules! create_runtime_api {
                     (weight, RuntimeBlockWeights::get().max_block)
                 }
 
```

Comment:
The implementation of `pallet_transaction_payment_rpc_runtime_api::TransactionPaymentCallApi` is missing. This was added between v0.9.26 and v0.9.29 and is a useful runtime call, since it allows us to query fee data using calls instead of extrinsics, as shown in the following example:
![image](https://user-images.githubusercontent.com/15378334/210827273-3fe7b714-2e0e-47df-a266-60f5550150f8.png)
![image](https://user-images.githubusercontent.com/15378334/210827441-3f807314-fe7b-49c8-9f95-a832d377c806.png)

Here is the patch:
[implement_TransactionPaymentCallApi_v0929.zip](https://github.com/zeitgeistpm/zeitgeist/files/10353760/implement_TransactionPaymentCallApi_v0929.zip)